Library/Formula/elasticsearch.rb

```diff
@@ -128,6 +106,6 @@ def plist; <<-EOS.undent
   end
 
   test do
-    system "#{bin}/plugin", "--list"
+    system "#{bin}/plugin", "list"
```

Comment:
Can we add another test that actually starts elasticsearch and checks it's working with `curl`?